source/transfer_list.rst

```diff
@@ -902,5 +903,128 @@ This manifest contains the SPMC attribute node consumed by the SPMD
      - hdr_size
      - Holds a SPMC manifest image in DT format.
 
+.. _dice_entries:
+
+Entries related to DICE
+^^^^^^^^^^^^^^^^^^^^^^^
+
+These entries hold [DICE]_ and [Open-DICE]_ information elements that needs to be transferred from one boot stage to the next.
+
+This includes the CDIs used for attestation (and optional sealing), as well as the accumulated certificate chain.
+
+The following entry types are currently defined:
+
+**CDI for attestation (XFERLIST_DICE_CDI_ATTEST)**
+
+This entry type holds the CDI described in [DICE]_.
+In [Open-DICE]_ it is referred to as "Attestation CDI".
+
+The CDI computed by the Sender's boot stage SHALL be added to the TL.
+The CDI received from the previous boot stage (i.e., DICE layer) MUST NOT be forwarded.
```

Comment:
I know it's described in the TCG docs, but shouldn't we also require the CDI user to clean the memory ?

Reply:
It certainly doesn't hurt, I agree.

Reply:
See if [0bb6617](https://github.com/FirmwareHandoff/firmware_handoff/pull/38/commits/0bb6617400214acf8a2fef3b81a49e074feac46d) works for you.